packages/imperative/src/config/cmd/set-secure/set-secure.handler.ts

```diff
@@ -9,8 +9,9 @@
 *
 */
 
+import * as JSONC from "comment-json";
 import { ICommandHandler, IHandlerParameters } from "../../../../../cmd";
-import { IConfigOpts } from "../../../../../config";
+import { IConfigOpts, Config } from "../../../../../config";
```

Comment:
Is Config used in this file?

Reply:
It's not anymore, and I believe the Config.INDENT property can be made private again.
Working on it!
Good catch!